server/mailet/mailets/src/test/java/org/apache/james/transport/matchers/InSpammerBlacklistTest.java

```diff
@@ -68,8 +68,12 @@ private Mail createMail(String remoteAddr) throws MessagingException {
     private void setupMatcher(String blacklist) throws MessagingException {
         matcher = new InSpammerBlacklist();
         matcher.setDNSService(setUpDNSServer());
-        FakeMailContext context = FakeMailContext.defaultContext();
-        FakeMatcherConfig mci = new FakeMatcherConfig("InSpammerBlacklist=" + blacklist, context);
+        FakeMatcherConfig mci = FakeMatcherConfig.builder()
+                .matcherName("InSpammerBlacklist")
+                .mailetContext(FakeMailContext.defaultContext())
```

Comment:
Maybe we should have FakeMailContext.defaultContext() being the default value for mailContext in FakeMatcherConfig. It would avoid a lot of duplication.